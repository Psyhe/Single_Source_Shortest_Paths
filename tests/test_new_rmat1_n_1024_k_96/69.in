1024 754 763
756 67 67
758 4 191
758 138 49
760 520 3
