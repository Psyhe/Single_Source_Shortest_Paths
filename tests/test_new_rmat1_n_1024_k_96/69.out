1e+18
1e+18
69
1e+18
54
1e+18
8
1e+18
1e+18
1e+18
